openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-video-util.js

```diff
@@ -211,7 +211,11 @@ var VideoUtil = (function() {
 	}
 	function _isEdge(_b) {
 		const b = _b || kurentoUtils.WebRtcPeer.browser;
-		return b.name === 'Edge';
+		return b.name === 'Edge' && "MSGestureEvent" in window;
+	}
+	function _isEdgeChromium(_b) {
+		const b = _b || kurentoUtils.WebRtcPeer.browser;
+		return b.name === 'Edge' && !("MSGestureEvent" in window);
```

Comment:
Why not to use `b.major > 80` here?

Reply:
According to wiki on Windows last EdgeHTML-based edge was v44.19041 and first chromium-based v79.0.309. But for example on android first chromium-based Edge is v42.0.2.3928 and latest is v45.03.4.4955.
As you can see there is overlap between  v42 and v44 across platforms only with Windows and android in mind, but there are also builds for macox and ios.
Are we want to support old EdgeHTML-based edge at all? Currently it is broken due to CSP bugs with strict-dynamic and inline-unsafe.

Reply:
> Are we want to support old EdgeHTML-based edge at all? Currently it is broken due to CSP bugs with strict-dynamic and inline-unsafe.

Audio/video most probably will not work in this browser as well
But I believe we have no chances to just ignore it's existence

Reply:
I checked old EdgeHTML-based edge (v42.17134.1098.0 to be precise, it is the latest one without insiders preview) by adding ability to disable CSP (https://github.com/CkNoSFeRaTU/openmeetings/commit/a9028a167b7f583b3d32c65d59d7e629a27cb728). It does indeed has completely broken transmission and receiving due to kurento-utils.js trying to call **RTCPeerConnection.addTransceiver**, but adapter.js doesn't have shim for that. Also from what I can tell this old edge is also requiring to additionally call **RTCPeerConnection.addIceCandidate(null)** after receiving all candidates.

Also during testing I found out that this edge by just staying in the room will pretty fast hammer KMS:
**org.kurento.client.internal.server.KurentoServerException: Reached KMS files limit: 819 (Code:40009, Type:null, Data: {"type":"NOT_ENOUGH_RESOURCES"})**
And indead there are tons of hanging open UDP ports by KMS. This is probably due to [OPENMEETINGS-2355].

Reply:
Hopefully all Edge users (including corporate ones) can move to the latest Chromium-Edge ...